frontend/src/features/admin-form/create/builder-and-design/BuilderAndDesignDrawer/EditFieldDrawer/edit-fieldtype/EditEmail/EditEmail.tsx

```diff
@@ -81,19 +81,9 @@ export const EditEmail = ({ field }: EditEmailProps): JSX.Element => {
     },
```

Comment:
LGTM, can you please help to fix the last 2 lint comments? Thanks

Reply:
ok! in [0cca575](https://github.com/opengovsg/FormSG/pull/6497/commits/0cca57501cd24fa6b5deed7223efa04462af856a)